Assets/Scripts/PlayerAnimation.cs

```diff
@@ -0,0 +1,16 @@
+public enum PlayerAnimation
```

Comment:
these will allow us to select which animation should be used for a specific `interactable`. the `PlayerController` will later read these and run the corresponding animation.